pkg/registry/resource/resourceclaimtemplate/strategy_test.go

```diff
@@ -475,15 +477,19 @@ func TestClaimTemplateStrategyCreate(t *testing.T) {
 			fakeClient := fake.NewSimpleClientset(ns1, ns2)
 			mockNSClient := fakeClient.CoreV1().Namespaces()
 			featuregatetesting.SetFeatureGatesDuringTest(t, utilfeature.DefaultFeatureGate, featuregatetesting.FeatureOverrides{
-				features.DRAAdminAccess:     tc.adminAccess,
-				features.DRADeviceTaints:    tc.deviceTaints,
-				features.DRAPrioritizedList: tc.prioritizedList,
+				features.DRAAdminAccess:        tc.adminAccess,
+				features.DRADeviceTaints:       tc.deviceTaints,
+				features.DRAPrioritizedList:    tc.prioritizedList,
+				features.DRAConsumableCapacity: tc.consumableCapacity,
 			})
 			strategy := NewStrategy(mockNSClient)
 
 			obj := tc.obj.DeepCopy()
 			strategy.PrepareForCreate(ctx, obj)
 			if errs := strategy.Validate(ctx, obj); len(errs) != 0 {
+				if tc.expectValidationError == "" {
+					t.Fatalf("unexpected error: %v", errs[0])
+				}
```

Comment:
This is what triggered this whole change. I agree that it is a gap.

All of the `pkg/registry/resource/*/strategy_test.go` files have it (copy-and-paste...). I checked some other API groups. At least the ones I looked at were fine because they don't have test cases with validation errors and thus always expect no errors.

Can you also fix the other `pkg/registry/resource/*/strategy_test.go` files?

Reply:
@pohly Sure. But could you please help point me out the line? In devicetaintrule, I don't see the same issue where we pass the error message contains empty message condition when there is an unexpected error.